packages/playwright/config.ts

```diff
@@ -0,0 +1,25 @@
+const isDevMode = process.env['isDev']
```

Comment:
Can you check this? I think most people use that variable

```suggestion
const isDevMode = process.env['NODE_ENV'] === 'development'
```

Reply:
@yhtyyar94 or maybe it's better to use `process.env.CI`

Reply:
I will try CI